templates/zerver/help/reply-to-message.md

```diff
@@ -0,0 +1,58 @@
+# Replying to a Message
+There are three ways to reply to a message in Zulip.
+## Clicking on a message
+To reply to a message, click on the message.
```

Comment:
Also, you line widths are a bit unregular - try to make them as close to 76 characters as possible. Editors usually allow visual help for that, e.g. a vertical line in the UI.